### PR TITLE
DON-878 add tokens for month cell

### PR DESCRIPTION
**Pull request by LokmaneKrizou**
# Context
We require new tokens for Month Cell but the existing ones don't satisfy our requirements, In this PR we are introducing two new tokens that could be used for different use cases.

Token name | Day Primitive Token | Night Primitive Token
-- | -- | --
Surface / low-contrast | Grey 10 | Night grey 25
Other / Overlay | Black Alpha 20 | White Alpha 80

Screenshot 2024-11-07 at 17 25 57

+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkColors.kt [hunk 1]
```diff
@@ -34,6 +33,7 @@ public class BpkColors private constructor(
     public val line: Color,
     public val lineOnDark: Color,
     public val scrim: Color,
+    public val overlay: Color,
```
Comment (marianeum) [inline backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkColors.kt:36]: this file is auto-generated from foundations - you have to add the tokens in backpack foundations first: https://github.com/Skyscanner/backpack-foundations

Comment (LokmaneKrizou) [reply backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkColors.kt:36]: oh that's something new to me, lemme clone that repo and add them there then

Comment (marianeum) [reply backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkColors.kt:36]: give me a shout if you need some help!

**Comment by LokmaneKrizou** [comment]
cancelling this PR as we need to add the new tokens from backpack-foundation
PR of these changes has been merged now
https://github.com/Skyscanner/backpack-foundations/pull/376

**Pull request closed without merging**